File: Data Structures & Algorithms/binary-tree-inorder-traversal/submission-1.swift
/**
 * Definition for a binary tree node.
 * class TreeNode {
 *     var val: Int
 *     var left: TreeNode?
 *     var right: TreeNode?
 *     init(_ val: Int) {
 *         self.val = val
 *         self.left = nil
 *         self.right = nil
 *     }
 * }
 */

class Solution {
    func inorderTraversal(_ root: TreeNode?) -> [Int] {
        var result: [Int] = []

        func dfs(_ node: TreeNode?) {
            guard let node = node else { return }

            dfs(node.left)
            result.append(node.val)
            dfs(node.right)
        }

        dfs(root)
        return result
    }
}

File: Data Structures & Algorithms/score-of-a-string/submission-1.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        left = 0
        right = 1

        score = 0

        while right < len(s):
            score += abs(ord(s[left]) - ord(s[right]))
            left += 1
            right += 1
        
        return score